tests/System.IO.Compression.Tests/BrotliPerfomanceTests.cs

```diff
@@ -106,22 +105,22 @@ public enum CompressionType
             NormalData
         }
 
-        [Benchmark]
+        [Benchmark(InnerIterationCount = 10000)]
         [InlineData(CompressionType.CryptoRandom)]
         [InlineData(CompressionType.RepeatedSegments)]
         [InlineData(CompressionType.NormalData)]
-        public void Decompress(CompressionType type)
+        public void DecompressUsingStream(CompressionType type)
         {
             string testFilePath = CreateCompressedFile(type);
-            int bufferSize = 1024 * 32;
+            int bufferSize = 1024;
             int retCount = -1;
             var bytes = new byte[bufferSize];
             using (MemoryStream brStream = new MemoryStream(File.ReadAllBytes(testFilePath)))
                 foreach (var iteration in Benchmark.Iterations)
                     using (iteration.StartMeasurement())
-                        for (int i = 0; i < 10000; i++)
+                        for (int i = 0; i < Benchmark.InnerIterationCount; i++)
```

Comment:
@Vedin 
> For me it makes sense and looks like the https://github.com/dotnet/corefx/blob/master/src/System.IO.Compression/tests/Performance/Perf.DeflateStream.cs in corefx have a same bug, doesn’t it?

Please file an issue in corefx regarding this and label it area-System.IO.Compression (or if you like, submit a PR for the fix).